bin/MadGraph5_aMCatNLO/cards/production/2017/5TeV/dyellell012j_5f_NLO_FXFX/dyellell012j_5f_NLO_FXFX_proc_card.dat

```diff
@@ -0,0 +1,16 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
```

Comment:
please follow these instructions to cleanup the datacards
https://twiki.cern.ch/twiki/bin/view/CMS/Moriond18MC#Checks_to_the_datacards_BEFORE_T

Reply:
Sorry that skipped my attention Luca. I removed the commented/unuseful lines from both madgraph proc cards.

Reply:
remove p and j definitions